src/renderer/fonts/caching_shaper.rs

```diff
@@ -164,12 +165,13 @@ impl CachingShaper {
                 "Font width: {:.2}px (avg: {:.2}px)",
                 font_width, metrics.average_width
             );
-            self.fudge_factor = font_width.round() / font_width;
+            self.fudge_factor = font_width.round() / font_width / self.scale_factor;
```

Comment:
I’m not sure that using the scale factor here is correct, simce the width should already be scaled with that, and so should the pixels.

But I don’t have a development environment to test it with at the moment, so I might be wrong.

Reply:
After looking at it again, I still think I'm correct. Dividing by the scale factor here will just make the fudge factor wrong, unless the scale is close to 1.

What do you think @borisfaure? Why was it added?

Reply:
Setting `self.info` to none should be enough. And the move of the code ensures that the debug output is wrong.

Setting it to none is needed, since otherwise cached information will be used.

Reply:
I agree it looks weird. I'll do more tests on it with multiple screens and which should bring me different scale values